### ISSUE TITLE
Cannot read property 'subscribe' of undefined

### ISSUE THREAD

**Issue by schickling** [graphcool/graphql-playground#231]
@katopz commented on [Fri Oct 27 2017](https://github.com/graphcool/framework/issues/1064)

**Current behavior**
`Cannot read property 'subscribe' of undefined` throw after hit play button
![image](https://user-images.githubusercontent.com/97060/32116042-488a38a0-bb73-11e7-8886-c19c086246db.png)

**Reproduction**
Try to use subscriptions via playground, e.g.
```
subscription {
  Tracing(filter: {mutation_in: [CREATED]}) {
    node {
      id
    }
  }
}
```
hit play button and got error

**Expected behavior?**
Working subscription.

---

@ZaninAndrea commented on [Wed Nov 01 2017](https://github.com/graphcool/framework/issues/1064#issuecomment-341078523)

I'm having the same problem, as a temporary solution I switched from _view as admin_ to _view as everyone_ and it worked

**Comment by dzegarra** [comment]
I'm having the same problem. I can't switch views. I don't that option. 
This bug may be related to this other one: #257

**Comment by marktani** [comment]
This is still/again a problem. Now with this error message:

![image](https://user-images.githubusercontent.com/1780597/34574345-6f457ee8-f177-11e7-9196-bbb653640792.png)


my queries:

![image](https://user-images.githubusercontent.com/1780597/34574366-7cc30b12-f177-11e7-8a16-dbcc584931bd.png)

**Comment by marktani** [comment]
supposedly also fixed by #397.
